_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,83 @@
+
+
+## Unity
+
+We started out worried about how we would collaborate on the game, and in such a rapid-fire way that the time constraints demanded.
+We are all accustomed to Git and [Github](https://github.com/SylverStudios) at work and for personal projects, but since Unity controls so much of the entire development experience, including the translation from what you see in the GUI to what gets stored in which files, Git doesn't seem to be a great tool for it.
+Luckily, Unity comes with it's own built-in collaboration tool that has its own push, pull, conflict resolution, and cloud storage solutions.
+We tried it out and, voila! It worked seamlessly, and allowed for us to hit a very high collaborative velocity.
+
+There was an exciting learning curve to layout in Unity.
+We have experience with web design and layout in systems like React and Elm, but that experience didn't exactly translate to Unity menu & page layout.
+We were learning Unity at max velocity, so we had to make some compromises: like not learning The Right Way ™ to do layout in Unity, and accepting madness like hardcoding 20px padding in ridiculous functions called `MakeRoom()`.
+C# isn't to blame for that, nor is the Unity layout system, but sacrifices were necessary.
+I expect this to be far easier the next time around.
+
+
+## Design
+
+
+Art takes so much longer than you expect it to! We worked on the project for about 15 hours, and it's crazy how few assets came out of that.
+The effort was worth it though; we came away with a new appreciation for how much of the value and experience of a game come from just a few sprites of game pieces.
+The designer's role should encompass the entire feel of the game. With [The Ferryman]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %}) we had a lot of text, menus, and backgrounds which I didn't think needed "design," but very clearly did.
+Spending time thinking about layout and backgrounds is a low-cost/high-reward practice for beginner game design.
+
+Choosing a color pallet and letting Sam and RJ get at it just wasn't enough to give the in-game Bar the feeling that we wanted.
+Next time around, I think the design role will be a more holistic job, focused on putting less effort into more aspects of the game, instead of lots of effort into a handful of assets.
+
+
+## Timeline
+
+This part was awesome for us.
+Historically, we have had very soft deadlines, if any at all.
+The deadline was non-negotiable and it was short, we knew exactly how much effort we would be able to put in before losing our sanity.
+We prioritized ruthlessly, cutting out many "fun", but inessential features.
+We went through many rounds of scaling down the scope and at the end of Saturday morning, we had a plan for the minimal features that we could still consider a "game."
+The constraint of the jam was a huge benefit, because we _had_ to think thin, we set ourselves up for success and produced something usable.
```

Comment:
another run on sentencerino

Reply:
literally can't spell, can't believe you would point it out and then not let me keep my dignity by giving me the answer!